ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/RefactoringPanel.java

```diff
@@ -1090,17 +1147,45 @@ public void restoreDeviderLocation() {
         }
     }
 
+    public boolean isPreview()
+    {
+        return getPrefPreview();
+    }
+
+    public boolean isSelectedPreview()
```

Comment:
the state of the toggle button is persisted right away if I see this correctly, so  `isPreviewEnabled()` could be used instead and this method can be removed.

Reply:
I thought better and removed the `isPreview` method, since the stored value is used only for panel initialization. Renamed the `isSelectedPreview` method to `isPreviewEnabled`.